images/cilium/Dockerfile.dockerignore

```diff
@@ -28,4 +28,5 @@
 !/pkg
 !/plugins/cilium-cni
 !/proxylib
+!/operator
```

Comment:
As per my [previous comment](https://github.com/cilium/cilium/pull/14069/files#r526768402):

```suggestion
!/operator/option
```

Reply:
Also, please keep the list alphabetically sorted.

Reply:
thanks, my bad :face_with_thermometer: